### PR TITLE
WW-7 add basic params

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/WW-7

### CODE REVIEW

File: extensions/wikia/Qualaroo/QualarooHooks.class.php [hunk 1]
```diff
@@ -38,10 +38,13 @@ static public function onOasisSkinAssetGroupsBlocking( &$jsAssetGroups ) {
 	}
 
 	static public function onMakeGlobalVariablesScript ( &$vars, $outputPage ) {
-		global $wgNoExternals, $wgQualarooDevUrl, $wgDevelEnvironment, $wgQualarooUrl;
+		global $wgNoExternals, $wgQualarooDevUrl, $wgDevelEnvironment, $wgQualarooUrl, $wgUser;
 
 		if ( empty( $wgNoExternals ) && $outputPage->getSkin()->skinname == 'oasis' ) {
 			$vars['wgQualarooUrl'] = ($wgDevelEnvironment) ? $wgQualarooDevUrl : $wgQualarooUrl;
+			$vars['isContributor'] = $wgUser->getEditCount() > 0;
+			$vars['isCurrentWikiAdmin'] = in_array( $wgUser, ( new CommunityPageSpecialUsersModel() )->getAdmins() );
```
Comment (lukaszkonieczny) [inline extensions/wikia/Qualaroo/QualarooHooks.class.php]: I will not make this value depended on Community Page extension

File: extensions/wikia/Qualaroo/QualarooHooks.class.php [hunk 2]
```diff
@@ -38,10 +38,13 @@ static public function onOasisSkinAssetGroupsBlocking( &$jsAssetGroups ) {
...
+			$vars['fullVerticalName'] = (new MonetizationModuleHelper())->getWikiVertical();
```
Comment (lukaszkonieczny) [inline extensions/wikia/Qualaroo/QualarooHooks.class.php]: Same here. I will consider using https://github.com/Wikia/app/blob/dev/extensions/wikia/WikiFactory/Hubs/WikiFactoryHub.php#L206

File: extensions/wikia/Qualaroo/scripts/Qualaroo.js [hunk 3]
```diff
@@ -27,13 +27,18 @@
...
 		'verticalName': window.verticalName,
+		// all verticals
+		'fullVerticalName': window.fullVerticalName,
 		'visitorType': window.visitorType,
 		'isPowerUserAdmin': !!window.wikiaIsPowerUserAdmin,
 		'isPowerUserFrequent': !!window.wikiaIsPowerUserFrequent,
 		'isPowerUserLifetime': !!window.wikiaIsPowerUserLifetime,
 		'isLoggedIn': !!window.wgUserName,
-		'cpBenefitsModalShown': document.cookie.indexOf('cpBenefitsModalShown') > -1
+		'cpBenefitsModalShown': document.cookie.indexOf('cpBenefitsModalShown') > -1,
```
Comment (lukaszkonieczny) [inline extensions/wikia/Qualaroo/scripts/Qualaroo.js:41]: What do you think about changing `cpBenefitsModalShown` field to more descriptive one?

Comment (dianafa) [inline extensions/wikia/Qualaroo/scripts/Qualaroo.js:41]: `cpBenefitsModalShown` is already used in qualaroo tageting by Trevor and its the same name as cookie has, so IMO for being consistent I'd leave it as is

File: extensions/wikia/Qualaroo/QualarooHooks.class.php [hunk 4]
```diff
@@ -38,10 +38,14 @@ static public function onOasisSkinAssetGroupsBlocking( &$jsAssetGroups ) {
 	}
 
 	static public function onMakeGlobalVariablesScript ( &$vars, $outputPage ) {
-		global $wgNoExternals, $wgQualarooDevUrl, $wgDevelEnvironment, $wgQualarooUrl;
+		global $wgNoExternals, $wgQualarooDevUrl, $wgDevelEnvironment, $wgQualarooUrl, $wgUser, $wgCityId;
 
 		if ( empty( $wgNoExternals ) && $outputPage->getSkin()->skinname == 'oasis' ) {
 			$vars['wgQualarooUrl'] = ($wgDevelEnvironment) ? $wgQualarooDevUrl : $wgQualarooUrl;
+			$vars['isContributor'] = $wgUser->getEditCount() > 0;
+			$vars['isCurrentWikiAdmin'] = in_array( $wgUser->getId(), ( new WikiService() )->getWikiAdminIds() );
```
Comment (sebastianmarzjan) [inline extensions/wikia/Qualaroo/QualarooHooks.class.php]: I know getWikiAdminIds() employs cache, though I still recommend to check if this is something we can do on each request

File: extensions/wikia/Qualaroo/QualarooHooks.class.php [hunk 5]
```diff
@@ -38,10 +38,14 @@ static public function onOasisSkinAssetGroupsBlocking( &$jsAssetGroups ) {
...
+			$vars['fullVerticalName'] = (new WikiFactoryHub())->getWikiVertical( $wgCityId )['short'];
```
Comment (lukaszkonieczny) [inline extensions/wikia/Qualaroo/QualarooHooks.class.php]: spaces inside brackets

File: extensions/wikia/Qualaroo/scripts/Qualaroo.js (hunk 3 above)
Comment (lukaszkonieczny) [inline extensions/wikia/Qualaroo/scripts/Qualaroo.js:41]: :+1:

**Comment by lukaszkonieczny** [comment]
LGTM

**Pull request merged**